apps/studio/src/pages/[siteId]/dashboard/[resourceId]/index.tsx

```diff
@@ -3,7 +3,7 @@ import { Button } from '@opengovsg/design-system-react'
 import { HStack, VStack, Text } from '@chakra-ui/react'
 import { type NextPageWithLayout } from '~/lib/types'
 import _ from 'lodash'
```

Comment:
nit: out of scope of this PR, but try to avoid using full lodash import -- it bloats the bundle size (because treeshaking won't work on this).

prefer importing only what you need, and even then via its direct export instead of the barrel export.
aka

```ts
import { mapValues } from 'lodash' // 🚫 
import mapValues from 'lodash/mapValues' // ✅
```